plugins/wpe-headless/includes/settings/callbacks.php

```diff
@@ -286,6 +287,12 @@ function wpe_headless_display_enable_disable_fields() {
 			<input type="checkbox" id="enable_redirects" name="wpe_headless[enable_redirects]" value="1" <?php checked( $enable_redirects ); ?> />
 			<?php esc_html_e( 'Enable public route redirects', 'wpe-headless' ); ?>
 		</label>
+		<br />
+
+		<label for="enable_image_source">
+			<input type="checkbox" id="enable_image_source" name="wpe_headless[enable_image_source]" value="1" <?php checked( $enable_image_source ); ?> />
+			<?php esc_html_e( 'Enable media images served from WP endpoint', 'wpe-headless' ); ?>
```

Comment:
Would this be more clear for the checkbox label? cc @trevanhetzel 

```suggestion
			<?php esc_html_e( 'Use the WordPress domain for media URLs in post content', 'wpe-headless' ); ?>
```